jest.config.js

```diff
@@ -48,5 +49,7 @@ module.exports = {
     '/__tests__/',
     '/vendor/',
     '<rootDir>/Libraries/react-native/',
+    'android-patches',
```

Comment:
Looks like indentation is not consistent

Reply:
I'm not sure what you are referring to... each entry has 4 spaces as indent.